src/core/appgateway.tf

```diff
@@ -52,13 +52,13 @@ module "app_gw" {
 
   # Configure backends
   backends = {
-    apim = {
-      protocol     = "Https"
+    aks = {
+      protocol     = "Http"
       host         = trim(azurerm_dns_a_record.dns_a_api.fqdn, ".")
-      port         = 443
-      ip_addresses = module.apim.private_ip_addresses
-      probe        = "/status-0123456789abcdef"
-      probe_name   = "probe-apim"
+      port         = 80
+      ip_addresses = [var.reverse_proxy_ip]
+      probe        = "/spid/v1/metadata"
```

Comment:
probably we need a more agnostic probe
I tried
```
curl -k https://10.1.0.250:6443/readyz
```
but not works with aks

https://kubernetes.io/docs/reference/using-api/health-checks/

Reply:
I was neither able to find a better probe...
at this moment I would let asis

Reply:
I think @diegolagospagopa could provide some useful hint :)

Reply:
as shared, we will use the ms-product health check at this moment